doc/lua_api.md

```diff
@@ -8523,6 +8525,12 @@ Used by `minetest.register_abm`.
     -- combination of, these neighbors.
     -- If left out or empty, any neighbor will do.
     -- `group:groupname` can also be used here.
+    
+    without_neighbors = {"default:lava_source", "default:lava_flowing"},
+    -- Only apply `action` to nodes that don't have any of, or any
+    -- combination of, these neighbors.
+    -- If left out or empty, any no neighbor will prevent ABM call.
```

Comment:
> any no neighbor will prevent ABM call

??????

Reply:
doc updated to `If left out or empty, any no neighbor will prevent ABM call.`